rust/src/accuracy/tail_bounds/conservative_continuous_gaussian_tail_to_alpha.tex

```diff
@@ -37,11 +37,20 @@
 
 \begin{lemma}
     \label{erfc-err}
-    By exhaustive search in \texttt{erfc\_err\_analysis.py}, the implementation of \texttt{erfc} errs by no greater than one 32-bit float ulp.
+    The implementation of \texttt{erfc} differs from a conservatively rounded implementation by no greater than one 32-bit float ulp.
 \end{lemma}
 
+\begin{proof}
+    An exhaustive search is conducted in the following code, 
+    included here from the adjacent \texttt{erfc\_err\_analysis.py} file.
```

Comment:
I'm guessing a relative link isn't easy here, or else you would have done that? I'd have a slight preference to drop this line without a link: If someone is already looking at files in a directory, then they'll probably see `erfc_err_analysis.py` without our help, and if something isn't looking at a list of files, this will be confusing.